website/frontend/management/commands/scraper.py

```diff
@@ -198,12 +219,16 @@ def is_boring(old, new):
 
     def extra_canonical(s):
         """Ignore changes in whitespace or the date line"""
+        # This is fragile: depending on the text looking a particular way!
         nondate_portion = s.split('\n', 1)[1]
         return nondate_portion.split()
 
     if extra_canonical(oldu) == extra_canonical(newu):
         return True
 
+    # This seems kind of fragile.  Are we 100% sure that differences between
+    # these encodings are unimportant?  Also, how does this relate to non-latin
+    # text?
```

Comment:
When would it not be boring?

Note that boring changes are still stored, they just aren't displayed on the website.  Otherwise you get an issue where when a news site changes all their articles to use a different encoding scheme, the history of every article is polluted.